src/app/help-panel/utility/predictors-data-help/predictors-data-help.component.html

```diff
@@ -1,14 +1,16 @@
 <h4>Manage Predictors</h4>
-<p>These are independent variables that will be used for <strong>{{ selectedFacility.name }}'s</strong>
-    regression analysis. Predictors can be used to explain the change in your utility use.</p>
+<p>Manage and track independent variables that affect <strong>{{ selectedFacility.name }}'s</strong> utility consumption. Independent variables (or predictors) are used with regression analysis to model and normalize utility usage between years. Common predictors for manufacturing include production, cooling degree days, and heating degree days.</p>
+<p>For more information on selecting independent variables see the <a href="https://www.energy.gov/eere/amo/downloads/energy-intensity-baselining-and-tracking-guidance" target="_blank">Better Plants Energy Intensity Baselining and Tracking Guidance</a>.</p>
+
 <hr>
-<p>Use “Add/Edit Predictors” to manage the predictors tracked for your analysis.</p>
+<p>Use “Add/Edit Predictors” to add, rename, edit, or delete predictors from your analysis. Also use this to select which predictor is used as the production variable(s) for <strong>{{ selectedFacility.name }}'s</strong> utility analysis.</p>
 <hr>
-<p><span class="fa fa-check-circle-o"></span> Indicates the predictor has been selected as a production
-    item.
-</p>
+<p>Use “Import Data” to jump to the import data tab for easy upload of bulk data in a csv or excel format.</p>
 <hr>
-<p>Use “Add Predictor Entry” to input your monthly predictor data.</p>
+<p>Use “+ Add New Entry” to input new predictors for a new period.</p>
 <hr>
-<p>Use “Import Data” to bulk import Predictor data in csv or excel format.</p>
-<hr>
\ No newline at end of file
+<p>Use the pencil icon (<span class="fa fa-pencil"></span>) to edit the predictor entries in a row and the trashcan icon (<span class="fa fa-trash"></span>) to delete a row of predictor entries.</p>
+<hr>
+<p>Use the “Copy Table” option to send predictor data to your clipboard to be pasted into external documents.</p>
+<hr>
+<p><span class="fa fa-check-circle"></span> indicates the predictor has been selected as a production item.</p>
```

Comment:
"Use the toggle switch to indicate the predictor has been selected as a production item."